foreman-selinux-enable

```diff
@@ -17,23 +19,24 @@ is_redhat_6() {
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
-    # Load policy
-    /usr/sbin/semanage module -S $selinuxvariant \
-      -a /usr/share/selinux/${selinuxvariant}/foreman.pp.bz2
+    /usr/sbin/semanage port -E > $TMP_PORTS
```

Comment:
The redirection here doesn't seem to work on EL6 with policycoreutils-python-2.0.83-24.el6.

<pre>
[root@foreman6 ~]# rpm -Uvh foreman-selinux-1.12.0-0.develop.201605040716gitb1daca47.el6.noarch.rpm
Preparing...                ########################################### [100%]
   1:foreman-selinux        ########################################### [100%]
libsepol.context_from_record: type elasticsearch_port_t is not defined (No such file or directory).
libsepol.context_from_record: could not create context structure (Invalid argument).
libsepol.port_from_record: could not create port structure for range 9200:9300 (tcp) (Invalid argument).
libsepol.sepol_port_modify: could not load port range 9200 - 9300 (tcp) (Invalid argument).
libsemanage.dbase_policydb_modify: could not modify record value (Invalid argument).
libsemanage.semanage_base_merge_components: could not merge local modifications into policy (Invalid argument).
/usr/sbin/semanage: Could not commit semanage transaction
/usr/sbin/semanage: Port tcp/2375-2376 already defined
warning: %post(foreman-selinux-1.12.0-0.develop.201605040716gitb1daca47.el6.noarch) scriptlet failed, exit status 1
[ edited it and added a cat, uncommented the trap ]
# bash -x /usr/sbin/foreman-selinux-enable
+ set +e
++ mktemp -t foreman-selinux-enable.XXXXX
+ TMP_EXEC_BEFORE=/tmp/foreman-selinux-enable.56Ryo
++ mktemp -t foreman-selinux-enable.XXXXX
+ TMP_EXEC_AFTER=/tmp/foreman-selinux-enable.BhL8L
++ mktemp -t foreman-selinux-enable.XXXXX
+ TMP_PORTS=/tmp/foreman-selinux-enable.9VQCl
+ for selinuxvariant in targeted
+ /usr/sbin/semodule -s targeted -l
+ /usr/sbin/semanage port -E
+ cat /tmp/foreman-selinux-enable.9VQCl
+ sed s/-a/-d/g
+ grep elasticsearch_port_t /tmp/foreman-selinux-enable.9VQCl
+ echo 'boolean -m --on httpd_setrlimit'
+ grep -q docker_port_t /tmp/foreman-selinux-enable.9VQCl
+ echo 'port -a -t docker_port_t -p tcp 2375-2376'
+ is_redhat_6
++ rpm -q --whatprovides redhat-release --qf '%{version}'
+ test x6 = x6
+ grep -q foreman_osapi_compute_port_t /tmp/foreman-selinux-enable.9VQCl
+ echo 'port -a -t foreman_osapi_compute_port_t -p tcp 8774'
+ test -s /tmp/foreman-selinux-enable.56Ryo
+ /usr/sbin/semanage module -S targeted -a /usr/share/selinux/targeted/foreman.pp.bz2
libsepol.context_from_record: type elasticsearch_port_t is not defined (No such file or directory).
libsepol.context_from_record: could not create context structure (Invalid argument).
libsepol.port_from_record: could not create port structure for range 9200:9300 (tcp) (Invalid argument).
libsepol.sepol_port_modify: could not load port range 9200 - 9300 (tcp) (Invalid argument).
libsemanage.dbase_policydb_modify: could not modify record value (Invalid argument).
libsemanage.semanage_base_merge_components: could not merge local modifications into policy (Invalid argument).
/usr/sbin/semanage: Could not commit semanage transaction
+ test -s /tmp/foreman-selinux-enable.BhL8L
+ /usr/sbin/semanage -S targeted -i /tmp/foreman-selinux-enable.BhL8L
/usr/sbin/semanage: Port tcp/2375-2376 already defined
# ll /tmp/foreman-selinux-enable.9VQCl
-rw-------. 1 root root 0 May  4 08:44 /tmp/foreman-selinux-enable.9VQCl
# semanage port -E
port -a -t docker_port_t -p tcp 2375-2376
port -a -t elasticsearch_port_t -p tcp 9200-9300
# semanage port -E > /tmp/a
# ls -l /tmp/a
-rw-r--r--. 1 root root 0 May  4 08:46 /tmp/a
# semanage port -E | tee /tmp/a
port -a -t docker_port_t -p tcp 2375-2376
port -a -t elasticsearch_port_t -p tcp 9200-9300
# ls -l /tmp/a
-rw-r--r--. 1 root root 91 May  4 08:47 /tmp/a
</pre>